Dockerfile

```diff
@@ -39,8 +39,9 @@ COPY --from=unidata-tomcat-image ${CATALINA_HOME}/conf/web.xml ${CATALINA_HOME}/
 # Security enhanced server.xml
 COPY --from=unidata-tomcat-image ${CATALINA_HOME}/conf/server.xml ${CATALINA_HOME}/conf/
 
-ARG ERDDAP_VERSION=2.24
-ARG ERDDAP_CONTENT_URL=https://github.com/ERDDAP/erddap/releases/download/v$ERDDAP_VERSION/erddapContent.zip
+ARG ERDDAP_VERSION=2.25.1
+# ARG ERDDAP_CONTENT_URL=https://github.com/ERDDAP/erddap/releases/download/v$ERDDAP_VERSION/erddapContent.zip
+ARG ERDDAP_CONTENT_URL=https://github.com/ERDDAP/erddap/releases/download/v2.24/erddapContent.zip
```

Comment:
Don't merge this! Just testing...
That files is not in 2.25  and 2.25.1 but I'm assuming it did not change from 2.24. I only want to see if this will build with the env vars defined in my fork.

Reply:
> Looks like Chris John is OK with the addition of a "docker" string int footer version string:

The plan is to create an official ERDDAP docker image (https://github.com/ERDDAP/erddap/pull/225) and update the Axiom image to use the ERDDAP image as the base image and add custom functionality (`datasets.d` etc). I'm planning on adding the version string to the ERDDAP image, just need to pivot back to that effort.

For erddapContent, the versioning scheme has actually changed and the current release is 1.0.0.

https://github.com/ERDDAP/erddapContent/releases

So that should be tracked in a separate `ERDDAP_CONTENT_VERSION` var or similar...

Reply:
I think I captured that now.